public/pages/Dashboard/utils/helpers.js

```diff
@@ -45,8 +48,9 @@ export function addFirstAlert(firstAlert) {
     severity,
     start_time,
     last_notification_time,
-    monitor_name,
-    monitor_id,
+    monitor_name: monitor_name || workflow_name,
+    monitor_id: monitor_id || workflow_id,
```

Comment:
Would this influence the `undefined` `monitor_id` check on [line 168 of Dashboard.js](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/611/files#diff-90a4d66c6d6bea36116eea37f5ab884f0bb10fce8f7bc1997fc7c24cb01a9f05R168).

Reply:
Discussed offline. @amsiglan explained that it's fine for the monitorID to be either the `monitor_id` or the `workflow_id`; we just don't want the monitorId to be blank.